### ISSUE TITLE
composer install fails

### ISSUE THREAD

**Issue by tailorbuilt** [iAmKevinMcKee/teamsy#13]
after cloning the repo, I ran
`composer install`

but get the following error:

`➜  teamsy git:(master) composer install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run `composer update` or `composer update <package name>`.
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - Root composer.json requires php ^7.2.5 but your php version (8.0.9) does not satisfy that requirement.
  Problem 2
    - dragonmantank/cron-expression is locked to version v2.3.0 and an update of this package was not requested.
    - dragonmantank/cron-expression v2.3.0 requires php ^7.0 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 3
    - laravel-frontend-presets/tall is locked to version v1.7.4 and an update of this package was not requested.
    - laravel-frontend-presets/tall v1.7.4 requires php ^7.2.5 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 4
    - laravel/framework is locked to version v7.26.1 and an update of this package was not requested.
    - laravel/framework v7.26.1 requires php ^7.2.5 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 5
    - laravel/tinker is locked to version v2.4.2 and an update of this package was not requested.
    - laravel/tinker v2.4.2 requires php ^7.2 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 6
    - laravel/ui is locked to version v2.2.0 and an update of this package was not requested.
    - laravel/ui v2.2.0 requires php ^7.2.5 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 7
    - livewire/livewire is locked to version v1.3.3 and an update of this package was not requested.
    - livewire/livewire v1.3.3 requires php ^7.1.3 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 8
    - opis/closure is locked to version 3.5.6 and an update of this package was not requested.
    - opis/closure 3.5.6 requires php ^5.4 || ^7.0 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 9
    - paragonie/random_compat is locked to version v9.99.99 and an update of this package was not requested.
    - paragonie/random_compat v9.99.99 requires php ^7 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 10
    - facade/flare-client-php is locked to version 1.3.5 and an update of this package was not requested.
    - facade/flare-client-php 1.3.5 requires php ^7.1 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 11
    - facade/ignition is locked to version 2.3.6 and an update of this package was not requested.
    - facade/ignition 2.3.6 requires php ^7.2.5 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 12
    - facade/ignition-contracts is locked to version 1.0.1 and an update of this package was not requested.
    - facade/ignition-contracts 1.0.1 requires php ^7.1 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 13
    - filp/whoops is locked to version 2.7.3 and an update of this package was not requested.
    - filp/whoops 2.7.3 requires php ^5.5.9 || ^7.0 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 14
    - fzaninotto/faker is locked to version v1.9.1 and an update of this package was not requested.
    - fzaninotto/faker v1.9.1 requires php ^5.3.3 || ^7.0 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 15
    - maximebf/debugbar is locked to version v1.16.3 and an update of this package was not requested.
    - maximebf/debugbar v1.16.3 requires php ^7.1 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 16
    - nunomaduro/collision is locked to version v4.2.0 and an update of this package was not requested.
    - nunomaduro/collision v4.2.0 requires php ^7.2.5 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 17
    - phar-io/manifest is locked to version 1.0.3 and an update of this package was not requested.
    - phar-io/manifest 1.0.3 requires php ^5.6 || ^7.0 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 18
    - phar-io/version is locked to version 2.0.1 and an update of this package was not requested.
    - phar-io/version 2.0.1 requires php ^5.6 || ^7.0 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 19
    - phpspec/prophecy is locked to version 1.11.1 and an update of this package was not requested.
    - phpspec/prophecy 1.11.1 requires php ^7.2 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 20
    - phpunit/php-code-coverage is locked to version 7.0.10 and an update of this package was not requested.
    - phpunit/php-code-coverage 7.0.10 requires php ^7.2 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 21
    - phpunit/php-file-iterator is locked to version 2.0.2 and an update of this package was not requested.
    - phpunit/php-file-iterator 2.0.2 requires php ^7.1 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 22
    - phpunit/php-timer is locked to version 2.1.2 and an update of this package was not requested.
    - phpunit/php-timer 2.1.2 requires php ^7.1 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 23
    - phpunit/php-token-stream is locked to version 3.1.1 and an update of this package was not requested.
    - phpunit/php-token-stream 3.1.1 requires php ^7.1 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 24
    - phpunit/phpunit is locked to version 8.5.8 and an update of this package was not requested.
    - phpunit/phpunit 8.5.8 requires php ^7.2 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 25
    - sebastian/code-unit-reverse-lookup is locked to version 1.0.1 and an update of this package was not requested.
    - sebastian/code-unit-reverse-lookup 1.0.1 requires php ^5.6 || ^7.0 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 26
    - sebastian/comparator is locked to version 3.0.2 and an update of this package was not requested.
    - sebastian/comparator 3.0.2 requires php ^7.1 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 27
    - sebastian/diff is locked to version 3.0.2 and an update of this package was not requested.
    - sebastian/diff 3.0.2 requires php ^7.1 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 28
    - sebastian/environment is locked to version 4.2.3 and an update of this package was not requested.
    - sebastian/environment 4.2.3 requires php ^7.1 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 29
    - sebastian/exporter is locked to version 3.1.2 and an update of this package was not requested.
    - sebastian/exporter 3.1.2 requires php ^7.0 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 30
    - sebastian/global-state is locked to version 3.0.0 and an update of this package was not requested.
    - sebastian/global-state 3.0.0 requires php ^7.2 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 31
    - sebastian/object-enumerator is locked to version 3.0.3 and an update of this package was not requested.
    - sebastian/object-enumerator 3.0.3 requires php ^7.0 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 32
    - sebastian/object-reflector is locked to version 1.1.1 and an update of this package was not requested.
    - sebastian/object-reflector 1.1.1 requires php ^7.0 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 33
    - sebastian/recursion-context is locked to version 3.0.0 and an update of this package was not requested.
    - sebastian/recursion-context 3.0.0 requires php ^7.0 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 34
    - sebastian/resource-operations is locked to version 2.0.1 and an update of this package was not requested.
    - sebastian/resource-operations 2.0.1 requires php ^7.1 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 35
    - sebastian/type is locked to version 1.1.3 and an update of this package was not requested.
    - sebastian/type 1.1.3 requires php ^7.2 -> your php version (8.0.9) does not satisfy that requirement.
  Problem 36
    - paragonie/random_compat v9.99.99 requires php ^7 -> your php version (8.0.9) does not satisfy that requirement.
    - symfony/polyfill-php70 v1.18.1 requires paragonie/random_compat ~1.0|~2.0|~9.99 -> satisfiable by paragonie/random_compat[v9.99.99].
    - symfony/polyfill-php70 is locked to version v1.18.1 and an update of this package was not requested.`

so I ran 
`composer update`

and get this:

`Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires php ^7.2.5 but your php version (8.0.9) does not satisfy that requirement.
  Problem 2
    - laravel-frontend-presets/tall[v1.7.0, ..., v1.7.4] require php ^7.2.5 -> your php version (8.0.9) does not satisfy that requirement.
    - Root composer.json requires laravel-frontend-presets/tall ^1.7 -> satisfiable by laravel-frontend-presets/tall[v1.7.0, ..., v1.7.4].`

So, I'm guessing php 8 is not supported?

**Comment by iAmKevinMcKee** [comment]
Correct. Although I believe it should be a simple update in the composer file as all the code should be compatible with PHP 8. Feel free to submit a PR.
